docs/validators/setup.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 2
+---
+
+# Setting Up a Node
+
+In this guide we will be setting up a ZetaChain Mainnet Beta node. For a testnet
+node, follow the same instructions, but replace some values (like chain ID and
+seed nodes) with the ones provided in corresponding tables.
+
+## Install the ZetaChain Node Binary
+
+First, download the binary and give it executable permissions. Make sure to
+replace the version (`v13.0.0`) with the latest version passed in a
+[governance software upgrade proposal](/validators/proposals) and the platform
+(`-darwin-amd64`) with the one you need.
+
+```
+wget https://github.com/zeta-chain/node/releases/download/v13.0.0/zetacored-darwin-amd64 -O /usr/local/bin/zetacored
+chmod a+x /usr/local/bin/zetacored
+```
+
+Check out the GitHub for
+[more information on releases](https://github.com/zeta-chain/node/releases).
+
+Check that you have the correct version of `zetacored` installed:
+
+```
+zetacored version
+```
+
+## Initialize the Data Directory
+
+| Network      | Chain ID         |
+| ------------ | ---------------- |
+| Mainnet Beta | zetachain_7000-1 |
+| Testnet      | athens_7001-1    |
+
+```
+zetacored init MONIKER --chain-id zetachain_7000-1
+```
+
+Replace `MONIKER` with a name for your node. By default the node will be
+initialized in `$HOME/.zetacored`, but you can customize this with the `--home`
+flag.
+
+## Fetch Config Files
+
+| Network      | Config Files                                                   |
+| ------------ | -------------------------------------------------------------- |
+| Mainnet Beta | https://github.com/zeta-chain/network-config/tree/main/mainnet |
+| Testnet      | https://github.com/zeta-chain/network-config/tree/main/athens3 |
+
+Fetch and save config files in the data directory:
+
+```
+wget https://raw.githubusercontent.com/zeta-chain/network-config/main/mainnet/genesis.json -O ~/.zetacored/config/genesis.json
+wget https://raw.githubusercontent.com/zeta-chain/network-config/main/mainnet/client.toml -O ~/.zetacored/config/client.toml
+wget https://raw.githubusercontent.com/zeta-chain/network-config/main/mainnet/config.toml -O ~/.zetacored/config/config.toml
+wget https://raw.githubusercontent.com/zeta-chain/network-config/main/mainnet/app.toml -O ~/.zetacored/config/app.toml
+```
+
+## Modify the Config Files
+
+Even though the config files you fetched in the previous step already contain
+all the required values common for all nodes, you will need to make some
+adjustments specific to your node.
+
+Get the correct external address for your machine:
+
+```
+curl -4 icanhazip.com
+
+109.68.188.70
+```
```

Comment:
This curl command will only work on machines which have an external IPv4 address, which is not a requirement. AWS is charging for IPv4 addresses now.

IPv6 only nodes are acceptable.

Reply:
3ee41c5

Reply:
https://github.com/zeta-chain/docs/pull/270/commits/3ee41c56065b39899f118209d101ad9afeb4a5a5